### DISCUSSION TITLE
%E6%96%AD%E8%A8%80%E7%9A%84%E8%AE%BE%E8%AE%A1.md/%E6%96%AD%E8%A8%80%E7%9A%84%E8%AE%BE%E8%AE%A1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#645] [Announcements]
# %E6%96%AD%E8%A8%80%E7%9A%84%E8%AE%BE%E8%AE%A1.md/%E6%96%AD%E8%A8%80%E7%9A%84%E8%AE%BE%E8%AE%A1

一、支持的断言方法 equal; not_equal; is_; is_not; is_in; isnotin; is_instance; isnotinstance; greater; greater_equal; less; less_equal; match; keyisexsit; keyisnot_exsit; 二、check支持的2种方式 ...

http://localhost:8081/%E6%96%AD%E8%A8%80%E7%9A%84%E8%AE%BE%E8%AE%A1.md/%E6%96%AD%E8%A8%80%E7%9A%84%E8%AE%BE%E8%AE%A1.html

**Comment by tylerjordan7276604** [comment]
123
